native-schema-registry/c/include/glue_schema_registry_error.h

```diff
@@ -0,0 +1,54 @@
+#ifndef NATIVE_SCHEMA_REGISTRY_GLUE_SCHEMA_REGISTRY_ERROR_H
+#define NATIVE_SCHEMA_REGISTRY_GLUE_SCHEMA_REGISTRY_ERROR_H
+
+#include <stdio.h>
+
+//Error codes are arbitrarily listed from 5000. No specific reason.
+#define ERR_CODE_INVALID_STATE 5000
+#define ERR_CODE_NULL_PARAMETERS 5001
+#define ERR_CODE_GRAALVM_INIT_EXCEPTION 5002
+#define ERR_CODE_GRAALVM_TEARDOWN_EXCEPTION 5003
+#define ERR_CODE_INVALID_PARAMETERS 5004
+#define ERR_CODE_RUNTIME_ERROR 5005
+
+//TODO: Improve error reporting to respect logging levels.
+#define log_warn(msg, code) fprintf(stderr, "WARN: %s, Code: %d\n", msg, code)
```

Comment:
there's a whole logger in aws-c-common

Reply:
Thanks for pointing it out. I will evaluate and take the logging part separately.